### DISCUSSION TITLE
Initial release of the chacha20-poly1305 crate

### DISCUSSION THREAD

**Discussion by nyonson** [rust-bitcoin/rust-bitcoin#3524] [General]
The chacha20_poly1305 crate has landed in rust-bitcoin and I would like to update the BIP324 library to use it so it can drop its internal implementation. I am not sure the process to release a new crate like this though. It doesn't have any dependency so I think can happen out of band with the larger refactors currently underway.

**Comment by tcharding** [comment]
@apoelstra does all the releases for crates in the `rust-bitcoin` repo. Just throw up a release tracking PR and we will ack and merge it then Andrew will cut the release for you.

We'll need a changelog file in the `chacah20_poly1305` directory. Version number is set correctly already so we don't need to touch that, also that means no update to the lock files needed. You can see the usual way we do the release tracking PRs by looking at any of these: https://github.com/rust-bitcoin/rust-bitcoin/pulls?q=is%3Apr+is%3Aclosed+release+tracking+PR

**Comment by nyonson** [comment]
Thanks @tcharding , PR opened here: https://github.com/rust-bitcoin/rust-bitcoin/pull/3525
